stix2validator/validator.py

```diff
@@ -127,13 +158,10 @@ class ValidationResults(BaseResults):
             otherwise.
 
     """
-    def __init__(self, is_valid=False, errors=None, fatal=None, warnings=None,
-                 fn=None):
-        super(ValidationResults, self).__init__(is_valid)
+    def __init__(self, is_valid=False, errors=None, warnings=None):
+        super(ObjectValidationResults, self).__init__(is_valid)
         self.errors = errors
-        self.fatal = fatal
```

Comment:
`fatal` didn't seem to be about validation.  It seemed to be about non-validation problems, e.g. I/O errors when reading files.  So I moved this to the new `FileValidationResults` class.